internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -282,6 +288,15 @@ func (s *LoadBalancedWebService) httpLoadBalancerTarget() (targetContainer *stri
 		targetPort = s.manifest.Sidecars[aws.StringValue(targetContainer)].Port
 	}
 
+	// Route load balancer traffic to the target_port if mentioned.
```

Comment:
Should we have returned after L288 before L289?

Reply:
> Should we have returned after L288 before L289?

The reason why we do not return after 288 and before 289 is because, say we have the following manifest, where the customer wants to expose a new port `82` of the sidecar container, then he can write a manifest like the following  - 

```
name: lbservice
type: Load Balanced Web Service

image:
  port: 80

http:
  path: '/'
  target_port: 82
  target_container: nginx

sidecars:
  nginx:
    image:
      port: 81

```
In that case, if we return after line 288, then we are making the assumption that the customer has not provided the target_port and wants to open a port that is mentioned in the `sidecar.image.port`. That is why we need to specifically check for `target_port` condition as well. 